dotnet/README.md

```diff
@@ -13,11 +15,13 @@ MOMENTO_AUTH_TOKEN=<YOUR AUTH TOKEN> dotnet run --project MomentoApplication
 Example Code: [MomentoApplication](MomentoExamples/MomentoApplication/Program.cs)
 
 ## Using the .NET SDK in your project
-SDK is built for target framework [.NET Standard 2.1 ](https://github.com/dotnet/standard/blob/master/docs/versions/netstandard2.1.md)
+
+SDK is built for target framework [.NET Standard 2.1](https://github.com/dotnet/standard/blob/v2.1.0/docs/versions/netstandard2.1.md)
```

Comment:
should this for target framework 6.0 now?

Reply:
@danielamiao - .NET Standard version 2.1 supports .NET and .NET Core version 6.0 which I believe what we updated in this pr. So no need to change the standard version.

https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-1

<img width="576" alt="image" src="https://user-images.githubusercontent.com/22686260/160486286-6db77cc6-1d81-41c7-8408-03cc51640f0e.png">

Reply:
great thanks for checking, i got confused with all the version numbers